apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/compute/functions/ServerInRegionToNodeMetadata.java

```diff
@@ -146,6 +147,11 @@ public NodeMetadata apply(ServerInRegion serverInRegion) {
 
    public static final Predicate<Address> isPrivateAddress = new Predicate<Address>() {
```

Comment:
Rename the predicate to make it clean that it looks for private IPs that are not floating (or leave it as-is and create a floating ip predicate and combine both to classify the addresses). Otherwise this can cause confusion when reading the code.